app/views/brexit_checker/_action_list.html.erb

```diff
@@ -18,45 +17,44 @@
           <%= action.title %>
         <% end %>
       </p>
+      <% if action.lead_time.present? %>
+        <div class="govuk-body govuk-!-font-weight-bold">
+          <div class="govuk-inset-text">
+            <%= action.lead_time %>
+          </div>
+        </div>
+      <% end %>
 
       <% if action.consequence.present? %>
-        <p class="govuk-body govuk-!-font-size-16">
+        <p class="govuk-body">
           <%= action.consequence %>
         </p>
       <% end %>
       <% if action.exception.present? %>
-        <p class="govuk-body govuk-!-font-size-16">
+        <p class="govuk-body">
           <%= action.exception %>
         </p>
       <% end %>
 
       <% if action.guidance_url.present? %>
-        <p class="govuk-body govuk-!-font-size-16">
-          <% prompt = action.guidance_prompt || t("brexit_checker.action_list.guidance_prompt") %>
-          <%= prompt %>:
+        <div class="govuk-body govuk-!-font-size-16"> 
+          <p>
+            <%= action.guidance_prompt || t("brexit_checker.action_list.guidance_prompt") %>:
+          </p>
+          <p>
             <a
               href="<%= action.guidance_url %>"
               class="govuk-link"
               data-module="track-click"
-              data-track-action="<%= audience_heading %> <%= action_index %> - Guidance"
+              data-track-action="<%= "#{analytics_group}#{index + 1}"%> - Guidance"
               data-track-category="brexit-checker-results"
               data-track-label="<%= action.guidance_url %>"
               data-ecommerce-row
               data-ecommerce-path="<%= action.guidance_path %>"
```

Comment:
These ecommerce attributes are correct, but we need to reinstate the parent ecommerce attributes. This is the PR when it was added: https://github.com/alphagov/finder-frontend/pull/1764/files#diff-7aa12fb0100b200d68270bac601a1dcdR8

Reply:
thanks, I'll cherry pick those, was a bit of a nightmare tracking down everything that had/hadn't been merged.

Reply:
Ok that's up! Will resolve these and we can check in on the heroku app

Reply:
@huwd Have had a look at the heroku app and it looks good to me 👍